share/spice/quandl/fundamentals/quandl_fundamentals.css

```diff
@@ -0,0 +1,74 @@
+/*
+    Document   : 
+    Created on : 2015-02-12 08:21 PM
+    Author     : brianrisk
+    Description: styles based on the 'Stocks' spice for visual consistency
+*/
+
+#quandl h1.quandl-header {
```

Comment:
I can do this, but this is not conducive to code re-use.

The plan is to make more IAs for various data sources on Quandl.  These IAs will use many of the same resources such as the logo, styles and handlebars.  As resources shared between IAs are not an option at the time, the backup plan is to simply copy these files into each Quandl IA share directory.

If we have to use a different namespace for each IA (zci--quandl_fundamentals, zci-qundl_home_values, etc.), this makes the Quandl IA code more difficult to manage -- especially as we develop more IAs.

Is what you propose a necessary fix, or could we make an exception with this?